diff --git a/dwm/config.h b/dwm/config.h
index 3ccd194..6dde2e3 100644
--- a/dwm/config.h
+++ b/dwm/config.h
@@ -110,7 +110,7 @@ static const char *bridown[]    = { "osd", "brightness", "10%-", NULL };
 
 static const Key keys[] = {
 	/* modifier                     key        function        argument */
-  { MODKEY,                       XK_u,      spawn,          {.v = lockcmd } },
+  { MODKEY,                       XK_i,      spawn,          {.v = lockcmd } },
   { MODKEY,                       XK_space,  spawn,          {.v = dmenucmd } },
   { MODKEY,                       XK_v,      spawn,          {.v = clipboardcmd } },
   { MODKEY,                       XK_q,      killclient,     {0} },
@@ -118,36 +118,36 @@ static const Key keys[] = {
 	{ MODKEY|ShiftMask,             XK_q,      spawn,          {.v = exitdwmcmd } },
 
   /* --- Navigation --- */
-	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } }, // Next window
-	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } }, // Prev window
+	{ MODKEY,                       XK_n,      focusstack,     {.i = +1 } }, // Next window
+	{ MODKEY,                       XK_e,      focusstack,     {.i = -1 } }, // Prev window
 	{ MODKEY,                       XK_m,      togglebar,      {0} },
  
   /* --- Resizing --- */
-	{ MODKEY,                       XK_h,      setmfact,       {.f = -0.05} }, // Shrink master
-	{ MODKEY,                       XK_l,      setmfact,       {.f = +0.05} }, // Grow master
+	{ MODKEY,                       XK_m,      setmfact,       {.f = -0.05} }, // Shrink master
+	{ MODKEY,                       XK_i,      setmfact,       {.f = +0.05} }, // Grow master
 	{ MODKEY,                       XK_Tab,    view,           {0} },
 
 	/* --- Master Area --- */
-	{ MODKEY,                       XK_b,      zoom,           {0} },        // Push window to master
-	{ MODKEY|ShiftMask,             XK_b,      incnmaster,     {.i = +1 } }, // More windows in master
-	{ MODKEY|ControlMask,           XK_b,      incnmaster,     {.i = -1 } }, // Fewer windows in master
+	{ MODKEY,                       XK_z,      zoom,           {0} },        // Push window to master
+	{ MODKEY|ShiftMask,             XK_z,      incnmaster,     {.i = +1 } }, // More windows in master
+	{ MODKEY|ControlMask,           XK_z,      incnmaster,     {.i = -1 } }, // Fewer windows in master
 
   /* --- Layouts --- */
-	{ MODKEY,                       XK_z,      setlayout,      {.v = &layouts[0]} },
-	{ MODKEY,                       XK_x,      setlayout,      {.v = &layouts[1]} },
-	{ MODKEY,                       XK_c,      setlayout,      {.v = &layouts[2]} },
-	{ MODKEY,                       XK_n,      setlayout,      {0} },
+	{ MODKEY,                       XK_x,      setlayout,      {.v = &layouts[0]} },
+	{ MODKEY,                       XK_c,      setlayout,      {.v = &layouts[1]} },
+	{ MODKEY,                       XK_d,      setlayout,      {.v = &layouts[2]} },
+	{ MODKEY,                       XK_k,      setlayout,      {0} },
   { MODKEY,                       XK_g,      togglefloating, {0} },
   { MODKEY,                       XK_a,      togglefullscr,  {0} },
   { MODKEY,                       XK_s,      togglesticky,   {0} },
 
   /* --- Vanity Gaps --- */
-  { MODKEY|Mod1Mask,              XK_h,      incrgaps,       {.i = +1 } },
-	{ MODKEY|Mod1Mask,              XK_l,      incrgaps,       {.i = -1 } },
-	{ MODKEY|Mod1Mask|ShiftMask,    XK_h,      incrogaps,      {.i = +1 } },
-	{ MODKEY|Mod1Mask|ShiftMask,    XK_l,      incrogaps,      {.i = -1 } },
-	{ MODKEY|Mod1Mask|ControlMask,  XK_h,      incrigaps,      {.i = +1 } },
-	{ MODKEY|Mod1Mask|ControlMask,  XK_l,      incrigaps,      {.i = -1 } },
+  { MODKEY|Mod1Mask,              XK_m,      incrgaps,       {.i = +1 } },
+	{ MODKEY|Mod1Mask,              XK_i,      incrgaps,       {.i = -1 } },
+	{ MODKEY|Mod1Mask|ShiftMask,    XK_m,      incrogaps,      {.i = +1 } },
+	{ MODKEY|Mod1Mask|ShiftMask,    XK_i,      incrogaps,      {.i = -1 } },
+	{ MODKEY|Mod1Mask|ControlMask,  XK_m,      incrigaps,      {.i = +1 } },
+	{ MODKEY|Mod1Mask|ControlMask,  XK_i,      incrigaps,      {.i = -1 } },
 	{ MODKEY|Mod1Mask,              XK_0,      togglegaps,     {0} },
 	{ MODKEY|Mod1Mask|ShiftMask,    XK_0,      defaultgaps,    {0} },
 	{ MODKEY,                       XK_y,      incrihgaps,     {.i = +1 } },
diff --git a/st/config.h b/st/config.h
index 3898969..72b1f67 100644
--- a/st/config.h
+++ b/st/config.h
@@ -149,8 +149,8 @@ static Shortcut shortcuts[] = {
 	{ TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
 	{ ShiftMask,            XK_Page_Up,     kscrollup,      {.i = -1} },
     { ShiftMask,            XK_Page_Down,   kscrolldown,    {.i = -1} },
-	{ TERMMOD,              XK_K,           kscrollup,      {.i = 5} },
-	{ TERMMOD,              XK_J,           kscrolldown,    {.i = 5} },
+	{ TERMMOD,              XK_E,           kscrollup,      {.i = 5} },
+	{ TERMMOD,              XK_N,           kscrolldown,    {.i = 5} },
 };
 
 /*
diff --git a/startdwm b/startdwm
index 7e706cb..63f51df 100755
--- a/startdwm
+++ b/startdwm
@@ -8,4 +8,5 @@ export QT_QPA_PLATFORM="xcb"
 export QT_QPA_PLATFORMTHEME="qt6ct"
 export XDG_CURRENT_DESKTOP=AshDWM
 
+setxkbmap us -variant colemak_dh
 ~/.ashdwm/autostart.sh

